orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/ScmService.java

```diff
@@ -0,0 +1,32 @@
+package com.netflix.spinnaker.orca.igor;
+
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class ScmService {
```

Comment:
If this class is just a wrapper around the igor service (i.e. function calls are just passthroughs to igor), do you think it's worth it to have this class? Are you planning on adding more complex logic here in the future?

Reply:
Like I mentioned earlier, this started as I looked into the existing implementation of `BuildService` (which is also just a pass-through), but seemed to exist to abstract the fact that it's igor under the covers into a more human-friendly, purpose-oriented class name. My thinking was that, if I was to just use `IgorService`, then to be consistent we should change everywhere that's using `BuildService`, and I felt that was way beyond the scope of this PR and didn't actually buy us anything.